src/components/Form.js

```diff
@@ -1,7 +1,9 @@
 import AddBtn from './AddBtn';
 
 const InputItem = () => (
-  <div>
+  <div className="footer">
+    <div className="horizontal-divider" />
+    <h1 className="form-title">ADD NEW BOOK</h1>
     <AddBtn />
   </div>
```

Comment:
- [ ] 👍 Good work on the styling of your webpage! However, I noticed that the contact form elements doesn't align horizontally with the element above it as it does in the design. To enhance your page's alignment, please consider updating your code to match the design as closely as possible. Your attention to detail is appreciated! 👏💻🎨

| Your | Design |
| --- | --- |
|  ![image](https://github.com/Surafels/Bookstore/assets/55840999/c81f0a62-d919-4dbd-bc36-6884d4c877c7)| ![image](https://github.com/Surafels/Bookstore/assets/55840999/794af588-cfcd-4146-bb83-2dad52f5da5f)|